### DISCUSSION TITLE
Antigen Retrieval: How to Choose the Best Method?

### DISCUSSION THREAD

**Discussion by CarolCKoh** [IBEXImagingCommunity/ibex_imaging_knowledge_base#153] [Q&A] [answered]
Hi everyone,

I am currently working on a project that involves antigen retrieval and need guidance on how to choose the best method. 

Here are some details about my project:

Sample Type: Formalin-fixed, paraffin-embedded (FFPE) tissue samples
Staining Method: Immunofluorescence (IF)
I've read that different antigens may require different retrieval methods and that the pH of the retrieval buffer can also play a significant role. However, I am not sure how to determine the optimal conditions for my antigen.

Specifically, I would like to know:

Is there a guide or reference that outlines which antigen retrieval method is best for specific types of antigens? In the IBEX community, there is a guide, but sometimes it is controversial. For example, it sometimes suggests using AR6, sometimes AR9, and occasionally both (e.g., CD68, clone KP1). Does the choice of antigen retrieval method depend on the clone, the tissue, or the technique (immunohistochemistry vs. immunofluorescence)?
What is the best pH for antigen retrieval, and how does it vary with different antigens? How do I know if a pH of 6 or 9 is more suitable for my antigen, or should both be tested?
Is performing antigen retrieval in a water bath problematic? Are there recommended practices for this method?
Could anyone share their experiences or provide advice on these points? Any specific guidelines or protocols you recommend would be greatly appreciated.

Thank you in advance for your help!

**Comment by radtkea** [comment, marked as answer]
Hi CarolCKoh!

Thank you so much for your question! Please excuse my late reply. I have been traveling a lot recently and I wanted to provide a helpful response to you.

1. I admire your thoroughness, reading the literature and searching our knowledge base:)
2. You are correct! The pH of the antigen retrieval buffer _does_ influence antibody labeling. Additionally, the amount of heat, duration of heating, and chemical composition of the retrieval buffers all influence HIER. [Here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7604828/) is an excellent article that answers many of your questions and provides example protocols for different antigen retrieval devices.
3. Some vendors provide details about the antigen retrieval conditions for a particular antibody. This is so helpful! See all the captions in the associated "Images" for this [antibody](https://www.abcam.com/products/primary-antibodies/cd3-epsilon-antibody-sp7-ab16669.html). From this datasheet, I would conclude that pH 6 or 9 would work.
4. [Here](https://www.novusbio.com/products/ki67-mki67-antibody_nb500-170) is an another example of critical application notes provided by a vendor. This particular antibody requires high heat and pH 6. We have confirmed this result.
5. Lab experience (myself and others) has determined that FOXP3 (clone 236A/E7) needs a high pH (8-10) for antigen retrieval. 

Because antigen retrieval conditions can vary from antibody to antibody, it is important to adopt a "universal antigen retrieval" protocol for multiplexed experiments employing more than one antibody. We discuss this in our [review](https://www.nature.com/articles/s41592-021-01316-y). For this reason, we have adopted a dual antigen retrieval protocol: pH 6 for 30 minutes ER1 (AR9961) and pH 9 for 30 minutes ER2 (AR9640) using the Leica Bond. This was inspired by my friend Liz's [protocol.](https://www.protocols.io/view/cell-dive-platform-slide-clearing-and-antigen-retr-bp2l6b9yrgqe/v1). Historically, we had used pH 6 (AR6 buffer and a water bath). 

Recommendations:
1. Decide on a universal antigen retrieval protocol. I personally recommend a dual protocol with pH 6 and pH 9. 
2. Water bath: I have used it before and it's a very good low cost alternative. The article I shared above provides additional details. I found that I needed to increase the time to 40 minutes. Temperature was 95C. It is very important to use pre-heated buffers, e.g., place glass jars with buffer in water bath 1 hour before you place slides in the hot buffer. Be careful handling hot buffers. I burned myself:) 
3. Does your institute have access to a Leica Bond? We have a shared instrument that many people use and they just purchase reagents. I think their ER1 and ER2 buffers are excellent. Another alternative is this [NanoVIP](https://biogenex.com/product/nanovip/). I haven't used it but could connect you to people, if needed.
4. Another good option is a [pressure cooker](https://biocare.net/product/decloaking-chamber-nxgen-antigen-retrieval/) over a water bath. I know many people who love this device. My colleague [Kartik](https://lod.humanatlas.io/omap/19-palatine-tonsil-ibex/v1.0/) has used this device and their [Borg Decloaker buffer](https://biocare.net/product/borg-decloaker-rtu/). 

Happy to help if you have additional questions about antibodies, panels, and anything else. Just email me (andrea.radtke "at" nih.gov).

**Reply by CarolCKoh** [reply]
Hi Andrea,

Thank you so much for your thorough and detailed response! I really appreciate you taking the time to address my questions, especially given your busy travel schedule.

Your insights on the pH influence and the detailed recommendations for antigen retrieval protocols are incredibly helpful. I will definitely review the article you shared and consider adopting a dual protocol with pH 6 and pH 9 for my experiments.

Unfortunately, our institute does not currently have access to a Leica Bond, but I will look into the other alternatives you mentioned, like the NanoVIP and the pressure cooker. Your suggestion about using pre-heated buffers and the specific conditions for the water bath will also be very useful.

Thank you again for your valuable advice and for offering further assistance. I may take you up on that as I progress with my work.

Best regards,
Carol

**Comment by radtkea** [comment]
Greetings Carol,

So happy I could help answer your questions!

We just encountered this issue in the lab with a visiting fellow from Brazil. None of the antibodies he tested worked on his samples with pH 9 alone but many worked after dual antigen retrieval.

Do you have a set up to do manual deparaffinization? We provide details on the materials and method for FFPE samples in [Part B of Sample Preparation](https://www.nature.com/articles/s41596-021-00644-9#Sec29). You will note that I neglected to include the critical detail of "pre-heat buffers in water bath" before doing antigen retrieval. Additionally, I was only doing antigen retrieval with pH 6 buffer. I have learned along the way:)

I found this interesting protocol that uses a [$130 Instant Pot](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10265304/)! It looks very promising and they also report the antigen retrieval buffers they use and many antibodies for mouse FFPE tissues.

Yes, please reach out if you have questions/need help with reagents.

Best of luck!
A

**Reply by lihongshi2023** [reply]
Greetings Carol, 
I'd like to echo all of Andrea's great suggestions. In the past, I have done many histology works with FFPE tissue, every time when I got a new antibody, I'd like to test both PH9 and PH6 to see which method works better (I'd like to remind you sometimes the vendors of the antibodies may not test the antibodies in house, so their recommendations about AR solutions may not always work the best); also, if you happen to have a steamer, I figured out the steamer also works for me for AR: you need to pre-warm the AR solution, then heated the slides in the AR solutions for 45 minutes at 95 celsius. 
Good luck!
Grace
